src/index.html

```diff
@@ -6,9 +6,42 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="stylesheet"  href="https://fonts.googleapis.com/css2?family=Roboto">
```

Comment:
you've imported the font, but there is not font-weight, so you text does not match
should be something like this one

https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap